New Topic: Cloud Computing (#930)
Codecademy/docs: content/cloud-computing/concepts/types-of-cloud/types-of-cloud.md

---
Title: 'Types of Cloud'
Description: 'There are three types of cloud computing implementations.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Cloud Computing'
  - 'Networking'
CatalogContent:
  - 'learn/foundations-of-cloud-computing'
  - 'paths/back-end-engineer-career-path'
---

> There are three types of cloud computing implementations.

Reviewer: ```suggestion
There are three **types** of cloud computing implementations:
```

> - **Public cloud:** Public clouds are owned and operated by third-party cloud service providers that deliver their computing resources like servers and storage over the Internet. Examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.
> - **Private cloud:** Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.
> - **Hybrid cloud:** Hybrid clouds are a combination of both private and public clouds. Data and applications are allowed to be shared between the private and public clouds. Allowing data and applications to move between private and public clouds provides businesses greater flexibility, more deployment options, and helps optimize existing infrastructure, security and compliance.

Reviewer: ```suggestion
## Public Cloud

Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.

## Private Cloud

Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.

## Hybrid Cloud

Hybrid clouds are a combination of both private and public clouds. Data and applications are allowed to be shared between the private and public clouds. Allowing data and applications to move between private and public clouds provides businesses greater flexibility, more deployment options, and helps optimize existing infrastructure, security, and compliance.
```

Should we also include a section about "Multicloud" services, what it is, and how it differs from other types like hybrid cloud?

---
Title: 'Types of Cloud'
Description: 'There are three types of cloud computing implementations.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Cloud Computing'
  - 'Networking'
CatalogContent:
  - 'learn/foundations-of-cloud-computing'
  - 'paths/back-end-engineer-career-path'
---

There are three **types** of cloud computing implementations:

## Public Cloud

> Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.

Reviewer 2: I prefer lists as bullet points rather than listing clauses in a sentence, but up to you.

Reviewer: Good spot!

Reviewer: ```suggestion
Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include the following:

- [Amazon Web Services (AWS)](https://aws.amazon.com/)
- [Google Cloud Platform (GCP)](https://cloud.google.com/gcp)
- [Microsoft Azure](https://azure.microsoft.com/en-us/)
```

## Private Cloud

Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.

## Hybrid Cloud

Hybrid clouds are a combination of both private and public clouds. Data and applications are allowed to be shared between the private and public clouds. Allowing data and applications to move between private and public clouds provides businesses greater flexibility, more deployment options, and helps optimize existing infrastructure, security, and compliance.

> There is also what is referred to as **multicloud** implimentations, which combine the above. It refers to any implementation that utilizes more than one cloud solution. While it includes all hybrid clouds by definition, it also includes solutions that involve two or more public clouds, or two or more private clouds.

Reviewer 2: Should have its own header. Also, the use of bold text is inconsistent in this article.

Reviewer 2: Final sentence can remove "by definition"

Reviewer: ```suggestion
## Multicloud

Multicloud is a combination of the above, referring to any implementation that utilizes more than one cloud solution. While it includes all hybrid clouds, multicloud also includes solutions that involve two or more public clouds, or two or more private clouds.
```

---

> Title: 'Types of Cloud'

Reviewer 2: This one looks good!

> Description: 'There are three types of cloud computing implementations.'

Reviewer: ```suggestion
Description: 'The types of cloud computing implementations include public, private, hybrid, and multicloud.'
```

What do we think about this possible `Description`? It is brief, and includes many of the major words/phrased used in the content.

Subjects:
  - 'Computer Science'
Tags:
  - 'Cloud Computing'
  - 'Networking'
CatalogContent:
  - 'learn/foundations-of-cloud-computing'
  - 'paths/back-end-engineer-career-path'
---

There are several **types of cloud** computing implementation:

## Public Cloud

Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include the following:

- [Amazon Web Services (AWS)](https://aws.amazon.com/)
- [Google Cloud Platform (GCP)](https://cloud.google.com/gcp)
- [Microsoft Azure](https://azure.microsoft.com/en-us/)

## Private Cloud

Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.

## Hybrid Cloud

> Hybrid clouds are a combination of both private and public clouds. Data and applications are allowed to be shared between the private and public clouds. Allowing data and applications to move between private and public clouds provides businesses greater flexibility, more deployment options, and helps optimize existing infrastructure, security, and compliance.

Reviewer 2: Final sentence is a run on, consider breaking up into a list or lists
